### ISSUE TITLE
Choosing .so suffix during installation

### ISSUE THREAD

**Issue by katie-maher** [maxmind/libmaxminddb#323]
Hi, I was just wondering, is there a way to choose the shared object suffix used on installation/configuration? 
When I run ./configure & make, I end up with a shared object "libmaxminddb.so.0.0.7". Ideally, I would like the suffix to match the package version - in my case "libmaxminddb.so.1.7.1". Is there an option to do this? 
P.S, I am running this on Ubuntu 20.04.

Thank you in advance :)

**Comment by oschwald** [comment]
No, this is not configurable. I believe using the versioning this way is actually [discouraged](https://www.gnu.org/software/libtool/manual/html_node/Updating-version-info.html#:~:text=Never%20try%20to,any%20other%20release.).
